calcom/cal.com
# feat: google meet no show

## What does this PR do?



- Fixes https://github.com/calcom/cal.com/issues/17061 (GitHub issue number)
- Fixes CAL-4501 (Linear issue number - should be visible at the bottom of the GitHub issue description)



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1) Connect google calendar and install google meet
2) Create a event type and add webhook events after hosts don't join google meet, after guests don't join google meet.
3)  Book a meeting and check Task Table

If you also want to test triggerHostNoShow then in L63 packages/features/bookings/lib/handleNewBooking/scheduleNoShowTriggers.ts add this code 

```

import { triggerHostNoShow } from "@calcom/features/tasker/tasks/triggerNoShow/triggerHostNoShow";

await triggerHostNoShow(
      JSON.stringify({
        triggerEvent: WebhookTriggerEvents.AFTER_HOSTS_GOOGLE_MEET_NO_SHOW,
        bookingId: booking.id,
        webhook: {
          ...subscribersHostsNoShowStarted[0],
          time: subscribersHostsNoShowStarted[0].time,
          timeUnit: subscribersHostsNoShowStarted[0].timeUnit,
        },
        destinationCalendar,
      })
    );

```
